hordak/migrations/0051_amount_to_debit_credit_data.py

```diff
@@ -0,0 +1,25 @@
+# Generated by Django 5.1b1 on 2024-07-01 05:12
+
+from django.db import migrations
+
+
+class Migration(migrations.Migration):
+    dependencies = [
+        ("hordak", "0050_amount_to_debit_credit_add_fields"),
+    ]
+
+    operations = [
+        # On an M2 Macbook Pro, this took 16 seconds on 600k rows.
+        # Therefore on 10M rows this may take around 5 minutes.
+        migrations.RunSQL(
+            """
+                UPDATE
+                    hordak_leg
+                SET
+                    credit = CASE WHEN amount > 0 THEN amount END,
+                    debit = CASE WHEN amount < 0 THEN amount END,
```

Comment:
Tried migrating our staging system in preps for production. Found the `debit` column was holding a negative value. Same with the views.

I believe it should be flipped?

`debit = CASE WHEN amount < 0 THEN -amount END,`

Reply:
Thank you for catching this! Fixed.